python/cog/types.py

```diff
@@ -200,11 +200,7 @@ def __repr__(self) -> str:
                 type(self).__name__, id(self), object.__getattribute__(self, "__url__")
             )
         else:
-            return "<{} at 0x{:x} wrapping {!r}>".format(
-                type(self).__name__,
-                id(self),
-                target,
-            )
+            return f"<{type(self).__name__} at 0x{id(self):x} wrapping {target!r}>"
```

Comment:
Was this done automatically by a formatter? Either way, it's probably not great to have unrelated changes in a file that isn't otherwise changed by the commit.